spec/support/models/user.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class User < ActiveRecord::Base
```

Comment:
I think we can delete this file. It's already in `spec/support/fixtures/user.rb` right?